camel-idea-plugin/src/main/java/org/apache/camel/idea/CamelProjectComponent.java

```diff
@@ -79,7 +79,8 @@ public void rootsChanged(ModuleRootEvent event) {
                     getCamelIdeaService(project).clearLibraries();
 
                     for (Module module : ModuleManager.getInstance(project).getModules()) {
-                        getCamelIdeaService(project).scanForCamelDependencies(module);
+                        getCamelIdeaService(project).scanForCamelVersionChange(project, module);
```

Comment:
maybe we should consider merge them together in one loop to avoid we are looping over the same elements multiple times